backends/dpdk/dpdkAsmOpt.h

```diff
@@ -280,15 +280,25 @@ class ShortenTokenLength : public Transform {
         return s;
     }
 
-    const IR::Node* preorder(IR::DpdkLearnStatement *ls) override{
+    const IR::Node* preorder(IR::DpdkLearnStatement *ls) override {
         ls->action = shortenString(ls->action);
         return ls;
     }
 
-    const IR::Node* preorder(IR::DpdkApplyStatement *as) override{
+    const IR::Node* preorder(IR::DpdkApplyStatement *as) override {
         as->table = shortenString(as->table);
         return as;
     }
+
+    const IR::Node* preorder(IR::DpdkJmpStatement *j) override {
```

Comment:
We are checking somewhere else that the def and use are matching for shortened names, right?

Reply:
JmpStatement(uses) and LabelStatement (def)is guaranteed.